### ISSUE TITLE
.alter() doesn't call wrapIdentifier

### ISSUE THREAD

**Issue by heisian** [tgriesser/knex#2606]
Initial migration with `wrapIdentifier` (`knexSnakeCaseMappers` from Objection.js) being used for snake_case to camelCase conversion:
```node
exports.up = (knex) => knex.schema.withSchema('admin').table('policyLogs', (t) => {
  t.string('actionableType')
})
// Results in admin.policy_logs.actionable_type being created
```

column-altering migration:
```node
exports.up = (knex) => knex.schema.withSchema('admin').table('policyLogs', (t) => {
  t.text('actionableType').alter()
})
```

resulting SQL/error:
```
Knex:warning - migration failed with error: alter table "admin"."policy_logs" alter column "actionableType" drop default - column "actionableType" of relation "policy_logs" does not exist
```

Passing in the snake-cased column name works:
```node
exports.up = (knex) => knex.schema.withSchema('admin').table('policyLogs', (t) => {
  t.text('actionable_type').alter()
})
```

**Comment by elhigu** [comment]
https://runkit.com/embed/m9gp3b02aef9

seems to fail at least with postgresql, mysql works, didn't test others right now.

**Comment by heisian** [comment]
@elhigu #2612 think i'm missing anything here?

**Comment by heisian** [comment]
Fixed by #2612
